requestlog/stackdriver.go

```diff
@@ -74,6 +74,7 @@ func (l *StackdriverLogger) log(ent *Entry) error {
 			Seconds int64 `json:"seconds"`
 			Nanos   int   `json:"nanos"`
 		} `json:"timestamp"`
+		TraceID string `json:"trace"`
```

Comment:
Unfortunately, this isn't a `LogEntry` object, this is the JSON data structure parsed by the [Fluent plugin](https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/master/lib/fluent/plugin/out_google_cloud.rb), so the `trace` key isn't directly recognized. (Would you mind adding a comment to that effect somewhere? This isn't an obvious property of this code.)

Glancing at the plugin's source, it appears that `logging.googleapis.com/trace` and `logging.googleapis.com/spanId` are the two keys we're interested in ([source](https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/f93046d92f7722db2794a042c3f2dde5df91a90b/lib/fluent/plugin/out_google_cloud.rb#L154-L155)).